docs/plugins.md

```diff
@@ -87,7 +87,7 @@ These plugins apply transformations to your code.
 - [es2015-block-scoping](/docs/plugins/transform-es2015-block-scoping/)
 - [es2015-classes](/docs/plugins/transform-es2015-classes/)
 - [es2015-computed-properties](/docs/plugins/transform-es2015-computed-properties/)
-- [es2015-constants](/docs/plugins/check-es2015-constants/)
```

Comment:
No sure what you mean: all other links point the transform plugin. Only for const/let the link goes to the syntax checker, which itself links to the transformer. This changes the link directly to the const transformer.